src/otx/algorithms/common/utils/task_adapt.py

```diff
@@ -99,8 +99,13 @@ def unwrap_dataset(dataset):
     :param dataset: dataset object, an instance of a dataset.
     :return: tuple of dataset object and int, the base dataset and the number of times to repeat the dataset.
     """
+    from otx.algorithms.detection.adapters.mmdet.datasets import ImageTilingDataset
```

Comment:
In environments that only use classification, mmdet will not be installed. This is where we expect to see import errors.